Eatery/Controllers/Eateries/CampusMenuViewController.swift

```diff
@@ -204,6 +204,16 @@ class CampusMenuViewController: ImageParallaxScrollViewController {
         pageViewController.didMove(toParentViewController: self)
 
         pageViewController.view.hero.modifiers = createHeroModifiers(.fade, .translate)
+
+        if let currentEvent = eatery.activeEvent(atExactly: Date()) {
+            for (index, viewController) in viewControllers.enumerated() {
+                if currentEvent.desc == "Lite Lunch", viewController.meal == "Lunch" {
+                    pageViewController.currentViewControllerIndex = index
+                } else if viewController.event?.desc == currentEvent.desc {
+                    pageViewController.currentViewControllerIndex = index
```

Comment:
Why is this an else if statement if it does the same thing?

Reply:
Thank you for pointing this out! I'll fix it